test/unit/cue-emitter-spec.js

```diff
@@ -262,5 +262,57 @@ describe('CueEmitter', function() {
 
       cue._onUpdate(1.1, 1.0);
     });
+
+    it('should emit events on seeking', function(done) {
```

Comment:
if it should emit on seeking really is the question for me. i deliberately made it so it doesn't - https://github.com/bbc/peaks.js/blob/9973a32d0cc91aa36d6c6f34cd36aed8857e90a3/src/main/cues/cue-emitter.js#L142 - likely the only change it needs to actually start emitting this stuff again (though the tests might need some more mocking of the transport)

my position is: it shouldn't emit on seeking. my interpretation of seeking is that the playhead didn't touch the material, and these events come from the playhead. the listener (not the emitter) must adjust its state about what region it's in, as soon as seeking has occurred. this is trivial for the listener which has access to the list of regions and new playhead position/time.

Reply:
Sorry for the delay on this. I have a working implementation that emits events on seek,